python/veles/db/backend.py

```diff
@@ -523,7 +523,8 @@ def list(self, parent, tags=frozenset(), pos_filter=PosFilter()):
         return {NodeID(bytes(x)) for x, in c.fetchall()}
 
     def begin(self):
-        assert not self.db.in_transaction
+        if six.PY3:
```

Comment:
It just happens to be the only part of this file that had problems running on py2, and I figured I would just fix it if it was that easy.  While it probably won't be useful for any practical purpose, it will increase py2 test coverage (once we have tracker tests) of other areas of the code - that's how I found the __ne__ issue, for one.